src/pages/search/search-popover.ts

```diff
@@ -86,8 +87,7 @@ import { Globals } from '../../providers/globals';
     <ion-item>
       <ion-label>{{ 'SEARCH_CATEGORY' | translate }}</ion-label>
       <ion-select [(ngModel)]="options.category" [multiple]="!options.astags">
-        <ion-option value="" *ngIf="options.astags">{{ 'SEARCH_ANYCAT' | translate }}</ion-option>
```

Comment:
Nuked this since it seemed redundant against the "All categories" option. Lemme know if there's a better way to handle this.

Reply:
[easy reference](https://forum.ionicframework.com/t/ion-select-with-select-all-option/62410)